shared/pingpong/accounts.go

```diff
@@ -82,186 +85,232 @@ func ensureAccounts(ac libgoal.Client, initCfg PpConfig) (accounts map[string]ui
 		fmt.Printf("Located Source Account: %s -> %v\n", cfg.SrcAccount, accounts[cfg.SrcAccount])
 	}
 
-	// If we have more accounts than requested, pick the top N (not including src)
-	if len(accounts) > int(cfg.NumPartAccounts+1) {
-		fmt.Printf("Finding the richest %d accounts to use for transacting\n", cfg.NumPartAccounts)
-		accounts = takeTopAccounts(accounts, cfg.NumPartAccounts, cfg.SrcAccount)
-	} else {
-		// Not enough accounts yet (or just enough).  Create more if needed
-		if len(accounts) != int(cfg.NumPartAccounts+1) {
-			fmt.Printf("Not enough accounts - creating %d more\n", int(cfg.NumPartAccounts+1)-len(accounts))
-		}
-		accounts, err = generateAccounts(ac, accounts, cfg.NumPartAccounts, wallet)
-		if err != nil {
-			return
+	if cfg.NumAsset == 0 {
```

Comment:
Could you explain why this check is needed?

Reply:
Added comments explain this.

Reply:
Thanks for the clarification!